movo_desktop/movo_viz/rviz/robot.rviz

```diff
@@ -69,97 +66,251 @@ Visualization Manager:
         Expand Link Details: false
         Expand Tree: false
         Link Tree Style: Links in Alphabetic Order
-        aux_battery_link:
+        base_chassis_link:
           Alpha: 1
           Show Axes: false
           Show Trail: false
           Value: true
-        base_chassis_link:
+        base_link:
+          Alpha: 1
+          Show Axes: false
+          Show Trail: false
+        front_laser_link:
           Alpha: 1
           Show Axes: false
           Show Trail: false
           Value: true
-        base_laser_link:
+        kinect2_ir_link:
+          Alpha: 1
+          Show Axes: false
+          Show Trail: false
+        kinect2_ir_optical_frame:
+          Alpha: 1
+          Show Axes: false
+          Show Trail: false
+        kinect2_link:
           Alpha: 1
           Show Axes: false
           Show Trail: false
           Value: true
-        base_link:
+        kinect2_rgb_link:
```

Comment:
I see there's a bunch of links related to the kinect. How dod you know to add all of these links?

Reply:
i just loaded the robot model which had the kinect. didnt your realsense changes get merged in?